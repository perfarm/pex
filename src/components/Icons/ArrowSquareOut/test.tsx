import React from 'react';

import { render } from '@testing-library/react';

import { ArrowSquareOut } from '.';

describe('ArrowSquareOut', () => {
  test('should match snapshot', () => {
    const { container } = render(<ArrowSquareOut />);

    expect(container).toMatchSnapshot();
  });
});
