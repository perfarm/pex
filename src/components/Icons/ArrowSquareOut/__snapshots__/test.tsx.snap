// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArrowSquareOut should match snapshot 1`] = `
<div>
  <svg
    class="PJLV"
    fill="none"
    height="18"
    viewBox="0 0 24 24"
    width="18"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      class="PJLV"
      d="M20.25 9.375L20.2493 3.75075L14.625 3.75M13.4973 10.5027L20.2473 3.75268M17.25 13.5V19.5C17.25 19.6989 17.171 19.8897 17.0303 20.0303C16.8897 20.171 16.6989 20.25 16.5 20.25H4.5C4.30109 20.25 4.11032 20.171 3.96967 20.0303C3.82902 19.8897 3.75 19.6989 3.75 19.5V7.5C3.75 7.30109 3.82902 7.11032 3.96967 6.96967C4.11032 6.82902 4.30109 6.75 4.5 6.75H10.5"
      stroke="#898989"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="1.5"
    />
  </svg>
</div>
`;
